bin/browsertime.js

```diff
@@ -49,7 +50,23 @@ function run(url, options) {
       (scriptsByCategory, userScripts) => merge(scriptsByCategory, userScripts));
   }
 
-  engine.start()
+let xvfb;
+if (options.xvfb) {
+    const width = options.chrome ? options.chrome.mobileEmulation.width : undefined;
+    const height = options.chrome ? options.chrome.mobileEmulation.height: undefined;
+
+    let widthAndHeight = (width && height) ? width + 'x' + height : options.viewPort;
+
+    xvfb = new Xvfb({
+        displayNum: '99',
```

Comment:
I think it means it will start start with 99 is that one is free? lets check ffmpeg config.